src/ExternalApi/Isite/Domain/Article.php

```diff
@@ -16,7 +16,7 @@ public function __construct(
         string $parentPid,
         ?string $shortSynopsis,
         string $brandingId,
-        string $image,
+        IsiteImage $image,
```

Comment:
If we want to be able to render pages with invalid images I think this should allow null
```suggestion
        ?IsiteImage $image,
```